### PR TITLE
This is the fizzbuzz exercise. It does print out the required output.…

### PR DESCRIPTION
**Pull request by mkdev2015**
… However, it does not pass the test.

there is an Assertion error that i tried my best to find a fix but utterly failed.

### CODE REVIEW

**Comment by barberboy** [comment]
Hi, Matilda!

I think the only issue with this is that the `for` loop should start at 1 instead of 0. The rest of it looks good!

File: fizzbuzz.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+for(var i= 0;i <= 100; i++){
```
Comment (barberboy) [inline fizzbuzz.js]: Per the assignment instructions, the `for` loop should start at 1 instead of 0.

**Review by barberboy — CHANGES_REQUESTED** [review]

**Comment by mkdev2015** [comment]
assignment completed with test passed

**Review by barberboy — APPROVED** [review]

**Comment by barberboy** [comment]
This is great, Matilda! 👍  💯

**Pull request closed without merging**